demos/kubeflow/README.md

```diff
@@ -35,6 +45,10 @@ $ juju config dex-auth static-password=admin
 
 The kubeflow dashboard will available at `$IPADDR.nip.io`
 
+```sh
```

Comment:
just a comment: did you used non-graphical vm that does not have access to browser?

Reply:
I think it depends on user's environment & network settings. I put command here just for those people who can't access the GUI. It's not for everyone but it still useful if someone don't know how to sshuttle multipass vm.